docs/cvl/builtin.md

```diff
@@ -1,10 +1,27 @@
 Built-in Rules
 ==============
+`Certora` Prover has built-in general-purpose rules targeted at finding known vulnerabilities.
```

Comment:
Replace with 
The `Certora` Prover has built-in general-purpose rules targeted at finding known vulnerabilities.

Reply:
changed